sauce/features/budget/quick-budget-warning/index.js

```diff
@@ -0,0 +1,33 @@
+import { Feature } from 'core/feature';
+import * as toolkitHelper from 'helpers/toolkit';
+
```

Comment:
Wanna add `toolkit/` to the imports to go along with your PR for the toolkit alias?